internal/db/schema/migrations/oss/postgres/22/02_wh_network_address_dimensions.up.sql

```diff
@@ -0,0 +1,88 @@
+begin;
+
+  -- wh_try_cast_inet returns either the provided text cast into inet or a null.
+  create function wh_try_cast_inet(text)
```

Comment:
can you add some [sqltests](https://github.com/hashicorp/boundary/blob/main/internal/db/sqltest/README.md) for this function?